src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -345,6 +346,7 @@ public function load(array $configs, ContainerBuilder $container): void
         }
 
         if ($this->readConfigEnabled('http_client', $container, $config['http_client'])) {
+            $this->readConfigEnabled('rate_limiter', $container, $config['rate_limiter']); // warmup config enabled
```

Comment:
what does the comment mean?

Reply:
It makes sure that `isInitializedConfigEnabled()` will work, see https://github.com/symfony/symfony/pull/53550/files#diff-e4f52d5033296421d02e360c8355aa1931bdfb0c12579f4815ea02b724322c0cR2529

Reply:
Maybe us this as comment instead?

Reply:
Done